src/plugins/intel_cpu/src/nodes/executors/acl/acl_utils.hpp

```diff
@@ -109,5 +110,26 @@ inline arm_compute::DataLayout getAclDataLayoutByMemoryDesc(MemoryDescCPtr desc)
     return arm_compute::DataLayout::UNKNOWN;
 }
 
+/**
+* @brief Return static mutex for lock ComputeLibrary configuration
+* @return static std::mutex
+*/
+static arm_compute::Mutex & getConfigurationMutex() {
+    static arm_compute::Mutex mtx_config;
+    return mtx_config;
+}
```

Comment:
Is this static declaration in the hider intentional? Do we really need a separate definition per each translation unit?

Reply:
@maxnick Its solve problem with acl configure, I removed all functions when we get new version of scheduler from ACL team (they are changing scheduling in ACL)